sdk/core/azure-core/src/credentials/credentials.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/core/credentials/credentials.hpp"
+
+using Azure::Core::Credentials::TokenCredential;
+
+namespace {
+std::string const DefaultCredentialName = "Custom Credential";
```

Comment:
Should it be named default credential name or custom credential name?

Reply:
The only case now when this returns `"Custom Credential"` is if customer have implemented `TokenCredential` and does not have the override for `GetCredentilName()`. I.e. shen it is a custom credential. Plus, using "default" will be confusing since we have `DefaultAzureCredential`.

Reply:
> Plus, using "default" will be confusing since we have `DefaultAzureCredential`.

Agreed.

Rename `DefaultCredentialName` to be something like `CustomCredentialName` or even `DefaultCustomCredentialName` to be clearer.

Reply:
Never mind, the named string is gone now, so the feedback is out-dated.

Reply:
Ah, I see what you meant, the variable name. Yes, I removed them all now, to not place them in either unnamed namespace or private static member.